asreview/feature_extraction/tfidf.py

```diff
@@ -36,7 +40,12 @@ def full_hyper_space(self):
         from hyperopt import hp
 
         hyper_space, hyper_choices = super(Tfidf, self).full_hyper_space()
+        hyper_choices.update({
+            "fex_stop_words": ["english", "none"]
```

Comment:
What downstream code needs the `"none"`? Can't we use `None`?

Reply:
Configuration files don't have a type, so there's no real way to distinguish "none" or None. In that particular line it might not break, but just mangled somewhere down the line. I think this is the safest way to do it right now.